test/e2e/run_test.go

```diff
@@ -719,4 +718,41 @@ USER mail`
 
 		Expect(session.OutputToString()).To(Not(ContainSubstring("/dev/shm type tmpfs (ro,")))
 	})
+
+	It("podman run --pull from local registry", func() {
```

Comment:
this is going to be expensive for time in testing.  is there anyway we might be able to just pull and old image and tag as latest?

Reply:
just thinking out loud here ...

Reply:
Problem is the initial pull will update it, and the way its written it will pull the source from whever the image came from (image.Names). I don't know if we can spoof better...

Reply:
Bottom line is we don't want podman buildah --pull and podman run --pull to have different meanings. If you want  a new meaning then call it update.
podman run --update 
Will update the image if it is in local storage or pull fresh copy if is not local.
Then don't have podman run --pull.